src/main/java/cn/nukkit/block/BlockNetherPortal.java

```diff
@@ -120,8 +120,8 @@ protected AxisAlignedBB recalculateBoundingBox() {
         return this;
     }
 
-    public static void spawnPortal(Position pos)   {
-        Level lvl = pos.level;
+    public static void spawnPortal(Position pos) {
+        Level lvl = pos.level; //TODO: This will generate part of the time, seems to be only when the chunk is populated
```

Comment:
Is this `TODO` fixed?

Reply:
Maybe related to #289?

Reply:
Nope, I completely forgot about it

Reply:
Might be beneficial to look at 0f92bd115e908d4b7c2d4c21a6bef3c1f87a715a based on its description